policyengine_us/variables/gov/states/ga/tax/deductions/ga_itemized_deductions.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class ga_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Georgia itemized deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://apps.dor.ga.gov/FillableForms/PDFViewer/Index?form=2022GA500",
+        "https://dor.georgia.gov/document/document/2022-it-511-individual-income-tax-booklet/download",
+        "https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=14c633e3-6517-481d-90d5-8e8340415643&pdistocdocslideraccess=true&config=00JAA1MDBlYzczZi1lYjFlLTQxMTgtYWE3OS02YTgyOGM2NWJlMDYKAFBvZENhdGFsb2feed0oM9qoQOMCSJFX5qkd&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A65D2-CDH3-CGX8-044N-00008-00&pdcomponentid=234187&pdtocnodeidentifier=ABWAALAADAAN&ecomp=k2vckkk&prid=a2b8e5c1-a075-4bcb-8484-2a0337552eb2",
+    )
+    defined_for = StateCode.GA
+
+    # GA itemized deduction does not account for mortgage expenses
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.irs.deductions
+        itm_deds = [
+            deduction
+            for deduction in p.itemized_deductions
+            if deduction not in ["salt_deduction"]
+        ]
+
```

Comment:
need to add back state and local taxes, [see note in itemized deductions](file:///Users/pavelmakarchuk/Downloads/2022%20IT-511_Individual_Income_Tax_%20Booklet%20(13).pdf) , there are examples in other state itemized deductions